src/15utility.js

```diff
@@ -339,7 +343,15 @@ var loadFile = utils.loadFile = function(path, asy, success, error) {
                 }
             }
         }
-        //*/
+	} else if(utils.isReactNative) {
+        /* If ReactNative */

```

Comment:
The problem with uglyfy2 is that having start comments within a section that is stripped from the browserversion is not supported. Just change to `// if reactiveNative` to make it work. 

Second input: As this is also stated in the `If` just above, I recommend ` /* If ReactNative */`gets removed